cookbooks/providers/application/providers/default.rb

```diff
@@ -1,5 +1,26 @@
 action :install do
 
+  if new_resource.letsencrypt
+    log %{Install certbot-auto...}
+    execute 'wget certbot-auto' do
+      user 'root'
+      command 'wget https://dl.eff.org/certbot-auto -O /root/certbot-auto && chmod a+x /root/certbot-auto && mv /root/certbot-auto /usr/local/bin/certbot-auto'
+    end
+
+    log %{Update certbot-auto...}
+    execute 'update certbot-auto' do
+      user 'root'
+      command 'certbot-auto -n config_changes'
+    end
+
+    log %{Install certbot-auto cron...}
+    cron 'certbot-auto renew' do
+      minute  0
+      hour    5
+      command 'certbot-auto renew --quiet --no-self-upgrade'
```

Comment:
Nice I didn't know this existed! But does it persist across reboots? That would be my only concern. I would prefer `systemd` which also has cronjob-like features. Because my servers reboot every now and then and I don't want to have to run chef first to kick up the cron again.